feathr-impl/src/main/scala/com/linkedin/feathr/offline/client/FeathrClient.scala

```diff
@@ -68,6 +68,15 @@ class FeathrClient private[offline] (sparkSession: SparkSession, featureGroups:
     SparkFeaturizedDataset(joinedDF, FeaturizedDatasetMetadata(header=Some(header)))
   }
 
+  /**
+   * APU which joins features and also returns any swallowed exception msgs.
+   */
+  @InternalApi
+  def joinFeaturesWithSwallowedExceptions(joinConfig: FeatureJoinConfig, obsData: SparkFeaturizedDataset,
+    jobContext: JoinJobContext = JoinJobContext()): (SparkFeaturizedDataset, String) = {
+    (joinFeatures(joinConfig, obsData, jobContext), SwallowedExceptionHandlerUtils.swallowedExceptionMsgs)
```

Comment:
I think SwallowedExceptions seems a bit ambiguous. Could we change it something like `MutedExceptions` or `SupressedExceptions`?

Reply:
Good suggestion, changed it to SuppressedExceptions.